lib/src/freshchat.dart

```diff
@@ -29,6 +29,7 @@ class FlutterFreshchat {
   static Future<bool> init({
     @required String appID,
     @required String appKey,
+    @required String domain,
```

Comment:
@rpandey91  we should set the `domain` default value to "msdk.freshchat.com" and not make it compulsory.

Reply:
Are you sure this is not gonna change?  Is that so, "msdk.freshchat.com" default value would need to add in both library i.e. android as well as ios. Correct me if I'm wrong

Thanks

Reply:
No, this their default (global) server and they have country-wise servers for India, Australia &, etc, and we don't need to set it individually for both Android & IOS we can just set it in flutter.